coordinator/cql/src/main/java/org/apache/cassandra/stargate/transport/internal/CQLMessageHandler.java

```diff
@@ -0,0 +1,392 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
```

Comment:
* what the hack is this? is this really from a `.class` file?
* we need stargate authors header as well?

Reply:
Also: since it does look like decompiled sources, would it be possible to instead get the sources from C*4.0 branch included? Or more fundamentally, since `cql` depends on `cassandra-all` 4.0.10, what are the reasons why those classes cannot be used? (I assume some are non-public or so).

Reply:
Refactored the class from 4.0.10 code base.